change/@fluentui-react-f8187229-7071-4c6a-be6e-1ea511a021e2.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 8 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28015&commitId=757f7a4b366c448938001929c4c3a9b779043220&env=prod)

✅ There was 0 screenshots added, 2 screenshots removed, 1039 screenshots unchanged, 0 screenshots with different dimensions and 6 screenshots with visible difference.


  <details><summary><b>unknown</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchColorPicker.Disabled.Focused.chromium.png|482| Changed|
|SwatchColorPicker.Disabled.Hovered.chromium.png|440| Changed|
|SwatchColorPicker.Disabled.Mousedown.chromium.png|437| Changed|
|SwatchColorPicker.Disabled.Selected and Focused.chromium.png|437| Changed|
|SwatchColorPicker.Disabled.Selected and Hovered.chromium.png|437| Changed|
|SwatchColorPicker.Disabled.default.chromium.png|479| Changed|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Removed|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv8"/>